tests/e2e/alltests.js

```diff
@@ -64,5 +63,14 @@ test('Test update user role', async t => {
         .contains('Roles updated successfully');
 });
 
+test('Test search by administrator', async t => {
+    await t
+        .click('#admin-event-log-link')
+        .typeText('#admin-id', TEST_CAFE_USER_ID)
+        .pressKey('enter')
+        .expect(Selector('html').textContent)
+        .contains('Test Cafe Cafe');
+});
+

```

Comment:
If we had more tests I'd say organize them in their own files, but this is the only test I've written for the Events view so it's not useful to go in a file by itself. I like what you did for the `createUser.js` tests though.